webrtc.html

```diff
@@ -5337,7 +5337,7 @@ <h4>
                         <p>
                           For each non-stopped
                           {{RTCRtpTransceiverDirection/"sendrecv"}} transceiver
-                          of [= transceiver kind =] <var>kind</var>, set
+                          of {{RTCRtpTransceiver/transceiver kind}} <var>kind</var>, set
```

Comment:
No problem with exporting this is a concept not an API, so we need to leave the [= =] syntax, or the links are formatted like they're code statements, which we don't want. See e.g. https://github.com/w3c/webrtc-pc/pull/2857 for how to do it.

Reply:
```suggestion
                          of [=RTCRtpTransceiver/transceiver kind=] <var>kind</var>, set
```